plasmapy/particles/decorators.py

```diff
@@ -165,7 +165,7 @@ def particle_input(
         'uncharged'}`` and the particle does not have charge information
         associated with it.
 
-    `~plasmapy/utils/AtomicError`
+    `~plasmapy/utils/ParticleError`
```

Comment:
```suggestion
    `~plasmapy.particles.excpetions.ParticleError`
```

Actually this whole "Raises" section uses slashes `/` instead of dots `.` and thus links are not properly created.  Update all these instances.